homeassistant/components/xiaomi_miio/number.py

```diff
@@ -68,8 +68,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     if model not in [MODEL_AIRHUMIDIFIER_CA4]:
         return
 
-    for feature in NUMBER_TYPES:
-        number = NUMBER_TYPES[feature]
+    for feature, number in NUMBER_TYPES.items():
```

Comment:
Oh, lol, `features` isn't even used.

```suggestion
    for number in NUMBER_TYPES.values():
```